### ISSUE TITLE
Rename default branch from "master" to "main"

### ISSUE THREAD

**Issue by cfarmer-fearless** [CDCgov/template#43]
**What type of help do you need?**

* [ ] Question
* [ ] New Repo
* [ ] Delete Repo
* [ ] User Membership (please make sure new members are familiar with the [CDC open practices](https://github.com/CDCgov/template/blob/master/open_practices.md#profile-setup) and set up their profile with name and org info to help people collaborate with them)
* [X ] Other

**Please describe how you'd like us to help.**
The new standard default branch name is "main".  New repositories created in GitHub use "main" as the default branch name.  Can we rename this to be consistent with GitHub?

**Comment by boris-ning-usds** [comment]
@cfarmer-fearless - is this for the template repository?

**Comment by boris-ning-usds** [comment]
If so, this was just completed.

**Comment by cfarmer-fearless** [comment]
Yes, it's for the template repo.  I see that main is the default branch now.  Will close this.  Thanks!
